integr8ly/tutorial-web-app
# INTLY-3286: Add os4 integreatly services links in the landing page

## Motivation
We should show and link other services that OS4 RHMI provisions similar to how we do it on OS3 on the landing page.

JIRA: https://issues.jboss.org/browse/INTLY-3286

## What
The changes in this PR adds links to the services which RHMI provisions in OS4 to the *Applications* section in the landing page. 

## How
- [x] Add a new param called provisionedServices to the `window.OPENSHIFT_CONFIG`.
   - This will be made available to the webapp deployment config as an env var called `INSTALLED_SERVICES`.
   - This is an object which contains information of the services provisioned by RHMI in the following format:
```
{ 
   "3scale":{ 
      "Host":"https://3scale.example.com",
      "Version":"0.0.1"
   },
   "codeready":{ 
      "Host":"https://codeready.example.com",
      "Version":"0.0.1"
   },
   "fuse-managed":{ 
      "Host":"https://syndesis.example.com",
      "Version":"0.0.1"
   },
   "launcher":{ 
      "Host":"https://launcher.example.com",
      "Version":"0.0.1"
   },
   "user-rhsso":{ 
      "Host":"https://sso.example.com",
      "Version":"0.0.1"
   }
}
```
- [x] Add the provisioned services to the list of services included in the *Applications* section in the landing page.

## Verification Steps
1. Run the webapp with the param `INSTALLED_SERVICES` defined with a value similar to the object above.
2. Ensure that all of the services included in the `INSTALLED_SERVICES` are shown in the *Applications* section in the landing page and has the correct links.

## Additional Notes
Related PR: https://github.com/integr8ly/integreatly-operator/pull/125
